### PR TITLE
Enable network share

### PR DESCRIPTION
**Pull request by deye9**
Changed SERVE_EXTERNALLY env variable to ENABLE_NETWORK_SHARE #1

### CODE REVIEW

File: framework/src/main/kotlin/dev/alpas/http/jetty/JettyServer.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ internal class JettyServer {
                 addCustomizer(ForwardedRequestCustomizer())
             }
             val connector = ServerConnector(server, HttpConnectionFactory(httpConfig)).apply {
-                if (!config.serveExternally) {
+                if (!config.enableNetworkShare) {
```
Comment (ashokgelal) [inline framework/src/main/kotlin/dev/alpas/http/jetty/JettyServer.kt]: Where is the corresponding config change changes?

**Comment by ashokgelal** [comment]
Do you mind updating the corresponding documentation as well? https://github.com/ashokgelal/alpasdev/blob/master/src/main/resources/docs/installation.md

**Comment by deye9** [comment]
PR has been raised.

**Comment by ashokgelal** [comment]
Thank you!

**Pull request merged**